README.md

```diff
@@ -9,13 +9,16 @@ in pure go/cgo, rather than cgo bindings for libsigar.
 
 ## Test drive
```

Comment:
Would you be willing to keep _both_ sets of "Test drive" instructions, and mention that the first set are for people who are using `go` versions older than whatever version requires the second set of instructions, and the second set for `go` versions equal to or greater than that version? (I assume that this version would be the one that introduced Go Modules, but I don't know for sure.)

(Or does `gosigar` no longer build with `go` versions that are older than the ones that will work with the first set of instructions?)

Reply:
The `go.mod` file has this:
https://github.com/cloudfoundry/gosigar/blob/af9ca463da970c167d3343a146dfd98be7afd808/go.mod#L3

Doesn't that make 1.19 the minimum version?

Reply:
Oh, yeah, that's a good point.

So, yeah, for now the change to the "Test Drive" instructions seems fine, as is. I'll add a thing to my personal TODO list to find out the _real_ minimum `go` version required to build and run this code, and fix up the README appropriately.